recipes/augur/meta.yaml

```diff
@@ -17,19 +17,20 @@ build:
 
 requirements:
   host:
-    - python >=3.6
+    - python >=3.7
     - pip
 
   run:
-    - python >=3.6
+    - python >=3.7
     - bcbio-gff >=0.6.0,<0.7
-    - biopython >=1.67,!=1.77,!=1.78
+    - biopython >=1.73,!=1.77,!=1.78
     - jsonschema >=3.0.0,<4
     - packaging >=19.2
     - pandas >=1.0.0,<2
     - treetime >=0.8,<0.9
     - xopen >=1.0.1,<2
     - cvxopt >=1.1.9,<2
+    - networkx >= 2.5, <3
```

Comment:
@victorlin I missed this in our review, but I think the space between `>=` and `2.5` is causing the test run to fail.

Reply:
Fixed: https://github.com/bioconda/bioconda-recipes/pull/32841/commits/5977f4033b46b3a4d1def7a33050e4ee8c6c11f2